### DISCUSSION TITLE
Just curiosity: Does it implemented a tiny nn framework in go supports Dense layer?

### DISCUSSION THREAD

**Discussion by lucasjinreal** [olivia-ai/olivia#168] [Q&A]
it seems whole neural network implemented in go include training process, why don't using popular framework like pytorch and transfer weights to go?
Also, what's the supported op coverage in olivia, such as convolution, lstm, transformers etc.

**Comment by hugolgst** [comment]
if I may I'll ask you the question the other way around: why should we use PyTorch and transfer weights to go?

I do not understand the second question, why should we implement convolution or transformers? Olivia is not an ML library.

**Reply by lucasjinreal** [reply]
。。。
